ndk/src/bitmap.rs

```diff
@@ -111,27 +137,87 @@ impl AndroidBitmap {
     }
 }
 
+/// Possible values for [`ffi::ANDROID_BITMAP_FLAGS_ALPHA_MASK`] within [`AndroidBitmapInfoFlags`]
+#[cfg(feature = "api-level-30")]
+#[derive(Clone, Copy, Debug)]
+pub enum AndroidBitmapInfoFlagsAlpha {
+    /// Pixel components are premultiplied by alpha.
+    #[doc(alias = "ANDROID_BITMAP_FLAGS_ALPHA_PREMUL")]
+    Premultiplied,
+    /// Pixels are opaque.
+    #[doc(alias = "ANDROID_BITMAP_FLAGS_ALPHA_OPAQUE")]
+    Opaque,
+    /// Pixel components are independent of alpha.
+    #[doc(alias = "ANDROID_BITMAP_FLAGS_ALPHA_UNPREMUL")]
+    Unpremultiplied,
+}
+
+/// Bitfield containing information about the bitmap.
+#[cfg(feature = "api-level-30")]
+#[derive(Clone, Copy)]
+#[repr(transparent)]
+pub struct AndroidBitmapInfoFlags(u32);
```

Comment:
Unsure if I should make this into a proper `bitflags`, there are currently only two valid values (the alpha mask, and the `IS_HARDWARE` flag).